### DISCUSSION TITLE
Budgets: Carry over unspent money

### DISCUSSION THREAD

**Discussion by taker218** [firefly-iii/firefly-iii#5762] [General]
Hello,

I don't know if this question was asked before, but I couldn't find it here on in the issues.
Would it be possible to add a setting to the budgets that unspent money gets carried over to the next month?

Currently I'm having one budget with the money I have for my renovation and so far I manually add the unspent amount from the last month to the next one.

Or is there another way how I could achieve this?

Best regards,
Thomas

**Comment by JC5** [comment]
https://docs.firefly-iii.org/firefly-iii/concepts/budgets/#rollover-add-an-amount-every-period

**Reply by taker218** [reply]
Ah I see.
The rollover one is not really what I was looking for, but the fixed amount one looks like it will be the one for my use case.

I just added a budget with a yearly fixed amount and this seems to do the job.
Thanks!
